packages/web-components/fast-components/docs/design/match-media-stylesheets.md

```diff
@@ -8,17 +8,41 @@ FAST exposes a mechanism to attach stylesheets conditionally based on a [MatchMe
 
 ### MatchMedia stylesheets
 
-`matchMediaStylesheetBehaviorFactory` can be used to construct a Behavior that will conditionally attach stylesheets based on the `matches` property of a [MediaQueryList](https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList).
+`matchMediaStylesheetBehavior` can be used to attach a stylesheet when a [MediaQueryList](https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList) matches and detach it when the query un-matches.
 
-__Example: Using the `matchMediaStylesheetBehaviorFactory`__
+__Example: Constructing the `MatchMediaStyleSheetBehavior`__
+```ts
```

Comment:
nit: this doc seems to be mix and matching the use of ts/js on the language selection for these codeblocks, should we just use either js or ts?